docker/Dockerfile

```diff
@@ -2,26 +2,38 @@
 FROM ubuntu:16.04
 
 # Update the base image
-RUN apt-get update
-RUN apt-get -y upgrade
-RUN apt-get -y dist-upgrade
-
-# Install Timesketch dependencies
-RUN apt-get -y install python-pip python-dev libffi-dev python-psycopg2
+RUN apt-get update && apt-get -y upgrade && apt-get -y dist-upgrade
+
+# Setup install environment and Timesketch dependencies
+RUN apt-get -y install apt-transport-https\
+                       curl\
+                       git\
+                       libffi-dev\
+                       lsb-release\
+                       python-dev\
+                       python-pip\
+                       python-psycopg2
+
+RUN curl -sS https://deb.nodesource.com/gpgkey/nodesource.gpg.key | apt-key add -
```

Comment:
Docker is for deployment, not development. We don't want/need yarn and nodeJS. THe bundle is part of the release to PyPi.

Reply:
Ok, after discussing this offline I can see the point of the docker stuff to run on HEAD. Future release changes (monthly) will address this and we can change then.